.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: continuous-integration
```

Comment:
Can you merge this new workflow with ours if it brings anything new?

Reply:
Done!

Reply:
Thanks 🙏